apps/web/app/(landing)/login/LoginForm.tsx

```diff
@@ -101,6 +201,14 @@ export function LoginForm() {
           <span className="ml-2">Sign in with Microsoft</span>
         </span>
       </Button>
+
+      <button
+        type="button"
+        onClick={handleSSOSignIn}
+        className="w-full rounded-md border border-slate-200 bg-white px-4 py-3 text-sm font-medium text-slate-700 shadow-sm transition-transform hover:scale-105 hover:bg-slate-50 hover:border-slate-300 focus:outline-none focus:ring-2 focus:ring-slate-500 focus:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50 disabled:bg-slate-50 dark:border-slate-700 dark:bg-slate-800 dark:text-slate-200 dark:hover:bg-slate-700 dark:hover:border-slate-600 dark:focus:ring-slate-400 dark:disabled:bg-slate-800"
+      >
+        Sign in with SAML/OIDC
```

Comment:
Can make this `Sign in with SSO`

And the button can be `Button` ghost/transparent/link (import our regular button component for this). The button shouldn't not take the user's attention as it doesn't matter to most people.

We don't need all the disclaimer stuff on the SSO form page btw